pymodbus/bit_read_message.py

```diff
@@ -155,7 +155,7 @@ def __init__(self, address=None, count=None, slave=0, **kwargs):
         """
         ReadBitsRequestBase.__init__(self, address, count, slave, **kwargs)
 
-    def execute(self, context):
+    async def execute(self, context):
```

Comment:
This code is as example used in the client as well, meaning it MUST work for both sync and async.